### ISSUE TITLE
[EN][Quest][MoltenFrontDaily]Into the fire

### ISSUE THREAD

**Issue by Thefakemccoy** [Atlantiss/BugTracker#5661]
Quest into the fire is bugged when you get to the end. There is 2 Pyrelords.
![wowscrnshot_010516_002253](https://cloud.githubusercontent.com/assets/12252441/12108195/e1ada976-b342-11e5-8e4c-7a31f4f397b1.jpg)

http://www.wowhead.com/quest=29206/into-the-fire

**Comment by Raknar** [comment]
I know about it. We have to remove double phase during that quest.

**Comment by eryczek1615** [comment]
quest coś długo się resetuje po godzinie czekania npc nie wrócił na swoje miejsce lecz stał tam gdzie się zabija Pyrelorda

**Comment by Luna-maria** [comment]
The npc to escort takes ages to spawn again (I retook the quest but that made no difference)
Waited for over 5 minutes to have the npc we gotta escort to spawn, for the npc to fail in making the flames go away and thereby killing whoever follows him, also making the quest impossible to do

**Comment by Aival94** [comment]
For me quest worked just fine and there were only 1 mob there to kill.

**Comment by Luna-maria** [comment]
There was 1 pirelord when I first did the quest, that got fixed earlier today
At the time of writing this though (revision 4786), when the npc we gotta escort 'finally' spawns, the flames around him don't go away, the quest is impossible to complete for me. There have been three restarts and a crash since I first did it on my other toon, something causes it to bug

(I even tried manually clearing my cache folder and it didn't help one bit)

The rate at how often the npc spawns and walks towards the pirelord needs to be decreased at any rate. Or if possible, trigger the npc to walk 10-20 seconds after the quest is accepted or something like that

**Comment by Thefakemccoy** [comment]
After the fix the fire wont go out by the druid.

**Comment by eryczek1615** [comment]
When we take daily quest Into the Fire the fire doesn't disappear. When we died on the Pyrelord and jump down after ress we haven't complete quest.

Sorry for my English

**Comment by Bastion** [comment]
no ja czekam już 2 godziny :D

**Comment by Luna-maria** [comment]
Aye... the druid doesn't put out the fire anymore (if he even spawns and walks there)

The pyrelord doesn't seem to spawn at all anymore now (http://www.wowhead.com/npc=52683/pyrelord) 
It might have something to do with Windcaller Nordrala standing next to the coil of ropes, looks like the script got stuck? 
Been waiting for over half an hour now, starting to wish for a server crash x.x (rev 4787, jan 9)

**Comment by Aival94** [comment]
For me yesterday quest did not worked as fires were staying on the ground but today even fires disappear.

As a minor but i think quest should be available after you complete your dailies before become available every day except from the [Druids of the Talon](http://www.wowhead.com/quest=29181/druids-of-the-talon) which is working like this now.More specific before become available should complete each day:

[Burn Victims](http://www.wowhead.com/quest=29138/burn-victims) and
[Hostile Elements](http://www.wowhead.com/quest=29179/hostile-elements) and
One of the following quests(depend of which is available each day):
[The Dogs of War](http://www.wowhead.com/quest=29304/the-dogs-of-war)
[The Harder They Fall](http://www.wowhead.com/quest=29141/the-harder-they-fall)
[Traitors Return](http://www.wowhead.com/quest=29142/traitors-return)
[Breach in the Defenses](http://www.wowhead.com/quest=29137/breach-in-the-defenses) and
One of the following quests(depend of which is available each day):
[Aggressive Growth](http://www.wowhead.com/quest=29139/aggressive-growth)
[Wisp Away](http://www.wowhead.com/quest=29143/wisp-away)

The same(about completing the dailies first) should be for the [The Forlorn Spire](http://www.wowhead.com/quest=29205/the-forlorn-spire) quest as well.

**Comment by Luna-maria** [comment]
until the script for this is re-written, can the spawntime on the pyrelord please be reduced so you don't have to wait ages to be able to do the quest?

**Comment by DuGalle** [comment]
Again this is broken NPC dont move dont cast the Pyrelord is not spawning the other npcs dont have quest for you down below. currently no way to open or finish this quest or those who follow up

**Comment by Karagrath** [comment]
@UP ...Any information about improvement this Q?  He's still bugged :(  Respawn this mob is incredibly long.

**Comment by Althin** [comment]
Still bugged. Druid don't go to clear the way from fire.

**Comment by karolnpc** [comment]
Bump Respawn of the druid lasts forever.

**Comment by karolnpc** [comment]
Quest doesnt work again. I guess it works only after server reset. In my opinion event should reset every 3 minutes or so.

**Comment by Elannsi** [comment]
@bump

**Comment by Hexenbane89** [comment]
bump, stil doesnt work

**Comment by Aival94** [comment]
Bump!
When quest change phase seems to send you to the end of this phase(when you have killed the mob) and the hall event have ended.
![wowscrnshot_051416_180054](https://cloud.githubusercontent.com/assets/10282971/15269164/ab268962-19ff-11e6-8e9d-9927760c10ae.jpg)

**Comment by Kubelecer** [comment]
As of 24/5 the quest is still bugged. The NPC refuses to move and put out the fire making the quest impossible to do.

**Comment by Aival94** [comment]
Furthermore event seems to be missing the [Flamewaker Assassin](http://www.wowhead.com/npc=53328/flamewaker-assassin) who are supposed to spawn to kill the NPC but on server only one is spawned and is at the end of the fires where the npc do not even go close to him to get aggro.

How it should work generally: [Into the Fire](https://www.youtube.com/watch?v=Gaq_LM-xFS4)

One more thing that might make you temporary fix the quest is that normally the [Pyrelord](http://www.wowhead.com/npc=52683/pyrelord) do not need the NPC event to be spawned as when you start the quest it should be spawned.
Without knowing how the quest is reseting you might want to reset it for rest except the one with the quest when the pyrelord get killed.

Proof about [Pyrelord](http://www.wowhead.com/npc=52683/pyrelord) spawned without the need of NPC event when he ask the druids to help you: [Pyrelord](https://www.youtube.com/watch?v=Gaq_LM-xFS4)

Note: Maybe someone want to update first post with all the bugs till now so make it easier for devs to see what is bugged without the need to read all these comments.

**Comment by Kubelecer** [comment]
Still bugged after update. Will this ever be fixed? It makes collecting marks very slow especially considering the vendors sell very useful things, if not mandatory such as the recipes for professions.

**Comment by Aival94** [comment]
Quests seems to get permanent bugged as even after a crash it does not reset(it keep the same phase every time).

**Comment by micha666** [comment]
Yep it's still bugged as it was. Waiting 30 minutes and nothing happens.

**Comment by Aequitass** [comment]
Bump. This quest should be on highest priority, so many ppl got stuck and can't move on further.
